index.html

```diff
@@ -83,6 +83,37 @@
 <section>
   <h2>Use Cases - Fundamental Features of a PWP</h2>
   <section>
+    <h2>Collection of Resources</h2>
+    <p>Publisher P works with multiple authors to create an anthology and uses
+      resources from different rights’ holders from different locations on the
+      web. Following the current practice on the Web, the publication consists
+      of many different resources (html, svg, css, etc.). P needs the
+      collection of all the resources as a unit to include it into its business
+      workflow. The publication must also be deposited to the national library
+      of as a legal deposit.</p>
```

Comment:
"… national library of as a legal deposit."

Either a word is missing or "of" is superfluous.